README.md

```diff
@@ -22,6 +22,7 @@ To record screenshots and compile them into a video using only one command, see
   * [Node Examples](#node-examples)
   * [Node API](#node-api)
 * [timesnap Modes](#modes)
+* [Control from page](#controlFromPage)
```

Comment:
```suggestion
```
With this revision, this will be an opt in behavior, so no need for a new section